README.md

```diff
@@ -1,11 +1,180 @@
 # proWES
 
-[![Apache License](https://img.shields.io/badge/license-Apache%202.0-blue.svg?style=flat)](http://www.apache.org/licenses/LICENSE-2.0)
-[![Build Status](https://travis-ci.com/elixir-cloud-aai/proWES.svg?branch=dev)](https://travis-ci.com/elixir-cloud-aai/proWES)
-[![Website](https://img.shields.io/website?url=https%3A%2F%2Fprotes.c03.k8s-popup.csc.fi%2Fga4gh%2Ftes%2Fv1%2Fui)](https://protes.c03.k8s-popup.csc.fi/ga4gh/tes/v1/ui/)
+[![license][badge-license]][badge-url-license]
+[![chat][badge-chat]][badge-url-chat]
+[![ci][badge-ci]][badge-url-ci]
 
-See
-[ELIXIR Cloud outline repository](https://github.com/elixir-europe/elixir-cloud-outline)
-for details.
+## Synopsis
 
-More info coming soon.
+proWES is a robust and scalable [Global Alliance for Genomics and Health
+(GA4GH)][res-ga4gh] [Workflow Execution Service (WES) API][res-ga4gh-wes] gateway
+that may play a pivotal role in augmenting the capabilities of your GA4GH Cloud
+ecosystem by offering flexible middleware injection for effectively federating
```

Comment:
> [...] for effectively federating [...]

From here on, this doesn't make sense - WES is not the same as TES. Maybe you could just say: 

> [...] for effective workflow deployment across a range of federated workflow management system deployments.